masq/src/terminal/terminal_interface.rs

```diff
@@ -24,9 +24,9 @@ mod test_cfg {
     pub use masq_lib::intentionally_blank;
 }
 
-//Unlike linefeed is designed to be used, I stick with using the system stdout handles for writing in instead of the custom handles provided from linefeed.
+//Unlike linefeed is designed to be used, I stick with using the system stdout handles for writing into them instead of the custom handles provided from linefeed.
```

Comment:
The name of the library is "linefeed," not "Linefeed," but "linefeed" can be confusing because it also refers to an ASCII control character. The confusion might be reduced by referring to it here as "the linefeed library" rather than just as "linefeed."